README.md

```
@@ -265,6 +266,46 @@ You can define a custom function that resolves the config.
+
+The validated output replaces `config`, so schema defaults, transforms and coercions are applied. This also means unknown keys are handled by **your schema**: most libraries strip them by default. To validate only a part of the config and keep the rest untouched, use a loose object:
+
+```js
+schema: z.looseObject({
+  port: z.coerce.number().default(3000),
+});
+```
+
+If validation fails, `loadConfig` throws an `Error` listing every issue, with the raw issues available on `error.cause`:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)